test/testchunkingng.cpp

```diff
@@ -600,7 +602,7 @@ private slots:
     // Test uploading large files (2.5GiB)
     void testVeryBigFiles() {
         FakeFolder fakeFolder{FileInfo::A12_B12_C12_S12()};
-        const qint64 size = 2.5 * 1024 * 1024 * 1024; // 2.5 GiB
+        const qint64 size = 1.5 * 1024 * 1024 * 1024; // 1.5 GiB
```

Comment:
Still bug but not a ssd stress test? Or what was the motivation.

Reply:
It's a good idea in general to explain why this size is ideal, since right now it looks like an arbitrary choice.

Reply:
added comment (2.5G > INT_MAX)